src/game-container/game-gfx-container/GfxContainer.svelte

```diff
@@ -26,29 +26,30 @@
             backContext     = initOverworld.initCanvas(0)      
             frontContext    = initOverworld.initCanvas(1)     
         }, 75 )
-
+        
         setTimeout( () => {
-            const playerCharacter = createCharInstance.getCharacter( charClass, charName, charClass )           
-
-            gameChar = playerCharacter
+            gameState.playerCharacter = createCharInstance.getCharacter( charClass, charName, charClass )           
```

Comment:
Op deze manier wordt het globale ding genaamd gameState telkens op allerlei plekken aangepast, als je dat in een aparte file doet met een private gameState die je alleen via methodes mag accessen dan wordt het overzichtelijker en makkelijker debuggen ook lijkt me

Reply:
Of zelfs een speciaal state js bestand maken om je algemene state in bij te houden